apps/server/src/shared/controller/transformer/sanitize-html.transformer.spec.ts

```diff
@@ -25,7 +25,7 @@ describe('SanitizeHtmlTransformer Decorator', () => {
 		contentRichTextCk5Simple!: string;
 	}
 
-	describe('when fully sanitizing an input string', () => {
+	describe('when sanitizing plain text', () => {
 		it('should remove all html', () => {
```

Comment:
in my opinion the two testcases somehow interfere... at least "remove all html" somehow contradict "not encode html entities" in my view, but maybe I did not understood fully?

Reply:
@MartinSchuhmacher The term "HTML entity" is related to special chatacters. This is opposed to DOM elements, see: https://developer.mozilla.org/en-US/docs/Glossary/Entity